recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+import glob
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+class AndreasbuhrCppCoroConan(ConanFile):
+    name = "andreasbuhr-cppcoro"
+    description = "A library of C++ coroutine abstractions for the coroutines TS"
+    topics = ("conan", "cpp", "async", "coroutines")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/andreasbuhr/cppcoro"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
```suggestion
    settings = "os", "compiler", "build_type", "arch"
    provides = "cppcoro"
```

This will force a conflict if both `cppcoro` and `andreasbuhr-cppcoro` are used in the same dependency graph.